index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my first portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;200;300;500&family=Roboto:ital@1&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <section class="header">
+         <h2><a href="#" class="logo">softkoda</a></h2>   
+        <nav class="nav-bar">
+            <ul>
+                <li><a href="#"><img src="images/Menu.png" alt=""></a></li>
+            </ul>
+        </nav>
+    </section>
```

Comment:
- [ ] [OPTIONAL] Please use the `header` tag instead of the section tag here, because it contains introductory content and a set of navigational links (which will be implemented in later milestones). 😃 
_This action is required_